hanadb_exporter/exporters/prometheus_metrics.py

```diff
@@ -40,6 +41,7 @@ def __init__(self):
         self.query = None
         self.metrics = []
         self.enabled = True
+        self.hana_version = '1.0.0.0'
```

Comment:
@arbulu89  Some questions here. The HANA database versioning has a different format, like `2.00.030.00.1522210459`. Will it be parsed correctly? I would say that the most relevant are the 3 first fields.

Reply:
During the comparison 2.00.030.00 would be greater than 2.0 for example.
The version comparisons are done well, so It wouldn't be a risk.